invisible_cities/cities/isidora_test.py

```diff
@@ -107,16 +103,19 @@ def test_isidora_run(config_tmpdir):
     nevt = fpp.run(nmax=nevts)
     assert nevt == nevts
 
+    # Check CWF and BLR waveforms match within 1% in the region of interest.
     eps = 1
+    peak_start = 4950
+    peak_end = 5150
     with tb.open_file(RWF_file, 'r') as h5in, tb.open_file(CWF_file, 'r') as h5out:
         pmtblr = h5in.root.RD.pmtblr
         pmtcwf = h5out.root.BLR.pmtcwf
         for event in range(nevt):
             BLR = pmtblr[event]
             CWF = pmtcwf[event]
             for pmt in range(len(BLR)):
-                diff = abs(np.sum(BLR[pmt][4950:5150]) -
-                           np.sum(CWF[pmt][4950:5150]))
+                diff = abs(np.sum(BLR[pmt][peak_start:peak_end]) -
+                           np.sum(CWF[pmt][peak_start:peak_end]))
                 diff = 100 * diff / np.sum(pmtblr[event][pmt])
                 assert diff < eps
```

Comment:
Do we really need a loop here? Can't this be expressed as an array operation and/or some numpy array assertion?